pkg/kubelet/pleg/generic.go

```diff
@@ -200,7 +200,7 @@ func (g *GenericPLEG) relist() {
 	}()
 
 	// Get all the pods.
-	podList, err := g.runtime.GetPods(true)
+	podList, err := g.runtime.GetPods(false)
```

Comment:
This change has bad side-effects.  It is intentional that all pods are fetched in this context so we can properly build the pleg.  The change should be that `updateRunningPodAndContainerMetrics` should do the desired filtering.